### PR TITLE
Update making-data-pipelines-in-r.md

### PR DESCRIPTION
**Pull request by Meghansaha**
cleaning up the abstract and updating affiliation

### CODE REVIEW

File: sessions/data-quality/making-data-pipelines-in-r.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ talk_title_short: Making Data Pipelines in R
...
 - name: Meghan S Harris
-  affiliation: University at Buffalo
+  affiliation: PCCTC @ Memorial Sloan Kettering
   url:
     webpage: https://www.thetidytrekker.com/
     twitter: https://twitter.com/meghansharris
     github: https://github.com/Meghansaha
     linkedin: https://www.linkedin.com/in/meghan-harris/
-    affiliation: https://medicine.buffalo.edu/departments/family-medicine/research.html
+    affiliation: **https://medicine.buffalo.edu/departments/family-medicine/research.html**
```
Comment (gadenbuie) [inline sessions/data-quality/making-data-pipelines-in-r.md:23]: Do you want to change or remove this link? This one isn't markdown compatible, so it needs to be just the URL. It's fine not to have a link here, too.

File: sessions/data-quality/making-data-pipelines-in-r.md [hunk 2]
```diff
@@ -47,8 +47,11 @@ from disbelief to amazement. R’s expansive capabilities can leave some feeling
...
-health department. Specifically, this talk will touch on the following topics:
+health department. Specifically, this talk will touch on the following concepts:
 
--Surveying Data Landscapes -File Structures -Saving Yourself with Data
-Validation -Modularizing Code -Connecting Scripts -Scaling for Dissemination
--Remaining Calm in Unfamiliar R Territories
+- Surveying Data Landscapes
+- File Structures 
+- Saving Yourself with Data Validation 
+- Modularizing Code and Connecting R Scripts -
```
Comment (gadenbuie) [inline sessions/data-quality/making-data-pipelines-in-r.md:55]:
```suggestion
- Modularizing Code and Connecting R Scripts
```

File: sessions/data-quality/making-data-pipelines-in-r.md (hunk 1 above)
Comment (Meghansaha) [reply sessions/data-quality/making-data-pipelines-in-r.md:23]: I'm sorry! Yeah I was just trying to change it as I don't work there anymore. We could remove it altogether.

Comment (gadenbuie) [reply sessions/data-quality/making-data-pipelines-in-r.md]: Sounds good, I'll take this URL out. And congrats on the new job! 🎉

**Review by gadenbuie — APPROVED** [review]
🚀

**Pull request merged**